examples/coagents-starter-praisonai-agents/agent-py/research/agents.py

```
@@ -0,0 +1,77 @@
+            expected_output=(
+                f"A list with 10 bullet points of the most relevant information about {topic}"
+            ),
+            agent=self.researcher
+        )
+    
+    def create_reporting_task(self, topic: str) -> Task:
+        """Create a reporting task"""
+        return Task(
+            name="Reporting Task",
+            description=(
+                "Review the context you got and expand each topic into a full section for a report. "
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)